languages/github-link-pt_PT.po

```diff
@@ -0,0 +1,35 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: GitHub Link\n"
```

Comment:
Wich ones? Whe whole header is generated by Poedit, the tool I use to generate .pot / .po files